app/components/filter/filter_component.rb

```diff
@@ -104,8 +104,12 @@ def custom_field_list_autocomplete_options(filter)
     end
 
     def custom_field_hierarchy_autocomplete_options(filter)
```

Comment:
:question: I was surprised by something here. Nothing in this `FilterComponent` makes me believe that the code you were touching was project-specific. Still, I checked and noticed that only the filters of projects were a flat list with a path-prefix until this point. Work package filters for the same kind of custom field were already displayed the way that you changed project custom fields to.

Do you happen to know, where the WP stuff is defined? Is there a good reason that those two are separated?